app/views/employers/plan_years/_publish_modal.html.erb

```diff
@@ -3,17 +3,30 @@
     <div class="modal-content">
       <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-        <h4 class="modal-title">Confirm Publish Plan Year</h4>
+        <h4 class="modal-title">Document Upload Required to Proceed</h4>
       </div>
       <div class="modal-body">
         <p>
-          As submitted, this application is ineligible for coverage under the <%= site_short_name %> exchange for reasons identified below. Click Cancel to go back and correct inaccurate information. If the information provided is accurate, you may click Publish to proceed. However, if you choose to proceed and the eligibility determination is deemed correct by <%= site_short_name %> officials, this organization may not again apply for coverage for 90 days, or until the reason for ineligibility changes, whichever is later.
+          Before you can complete your application, you will need to submit proof of your eligibility as a Massachusetts small business to shop through the Health Connector. You will need to submit this proof before you are able to publish your plan year. <%= link_to "View our list of documents that can be submitted as proof.", "https://www.mahealthconnector.org/business/business-resource-center/employer-verification-checklist" %> .
+          <br>
+          <ul>
+            <li>
+              To upload documentation of your eligibility, go to the Documents option on the left and follow instructions to upload the documentation of your eligibility.
+            </li>
+            <li>
+              Once you have uploaded your <%= link_to 'Documents', employers_employer_profile_path(@employer_profile.id, :tab=>'documents') %> you can return to this page and publish your plan year. 
+            </li>
+          </ul>
         </p>
-        <ul class="application-warnings"></ul>
+
       </div>
       <div class="modal-footer">
-        <%= link_to 'Force Publish', employers_employer_profile_plan_year_force_publish_path(@employer_profile,plan_year), method: :post, :class => 'btn btn-danger' %>
-        <% error_page = plan_year.application_eligibility_warnings[:primary_office_location].present? ? edit_employers_employer_profile_path(@employer_profile, plan_year_id: plan_year.id) : edit_employers_employer_profile_plan_year_path(@employer_profile,plan_year, publish: true) %>
+        <% if plan_year.documents_uploaded? %>
+            <%= link_to 'Go Back', employers_employer_profile_path(@employer_profile.id, :tab=>'benefits'), :class => 'btn btn-default' %>
+        <% else %>        
+                <%= link_to 'Force Publish', employers_employer_profile_plan_year_force_publish_path(@employer_profile,plan_year), method: :post, :class => 'btn btn-danger' %>
```

Comment:
if we don't want a force publish button why is this link named 'Force Publish' ?